src/tools/dart-doc.md

```diff
@@ -17,36 +18,29 @@ see the [documentation part of Effective Dart][effective doc].
   without errors.
 {{site.alert.end}}
 
-Run `dartdoc` from the root directory of your package. For example:
+Run `dart doc` from the root directory of your package. For example:
 
 ```terminal
 $ cd my_app
-$ dartdoc
+$ dart doc .
 Documenting my_app...
 ...
 Success! Docs generated into /Users/me/projects/my_app/doc/api
 ```
 
 By default, the documentation files are static HTML files,
-placed in the `doc/api` directory.
-To view the rendered documentation, you need an HTTP server.
+placed in the `doc/api` directory. You can create the
+files in a different directory with the `--output-dir` flag.
 
-Although `dartdoc` is in the Dart SDK,
-it's also available in the [dartdoc package.][]
-You might use the package when you need a programmatic interface
```

Comment:
Is it desired to deemphasize or not support the programmatic use case? I think it's quite useful to call out how `dart doc` is implemented and indicate its underlying implementation can also be used as a library. 

Perhaps just remove the mention of trying new features?

Reply:
I don't think this should be needed except in cases like validating a bug fix (in which case we can mention it in the bug).

When have you found it useful?

Reply:
I was originally thinking more about using the `dartdoc` API/library as a dependency. But taking a look at [packages using it as a dependency on pub](https://pub.dev/packages?q=dependency%3Adartdoc) there are very few users of it not as a dev dependency. 

So I suppose it is a rather niche and advanced functionality that likely doesn't need to be mentioned on the page.